modules/router/templates/assets_origin.conf.erb

```diff
@@ -38,16 +38,32 @@ server {
   add_header "Access-Control-Allow-Methods" "GET, OPTIONS";
   add_header "Access-Control-Allow-Headers" "origin, authorization";
 
-  <%- @asset_routes.each do |alias_path, vhost_name| -%>
-  location <%= alias_path %> {
-    proxy_set_header Host <%= vhost_name %>.<%= @app_domain %>;
-    proxy_pass <%= @enable_ssl ? 'https' : 'http' %>://<%= vhost_name %>.<%= @app_domain %>;
-  }
-
-  <%- end -%>
-
-  location / {
-    proxy_set_header Host static.<%= @app_domain %>;
-    proxy_pass <%= @enable_ssl ? 'https' : 'http' %>://static.<%= @app_domain %>;
-  }
+  <%- unless scope.lookupvar('::aws_migration') %>
```

Comment:
is there a reason your doing this as an `unless` rather than swapping the if/else blocks?